### DISCUSSION TITLE
Playback Issues

### DISCUSSION THREAD

**Discussion by jcostel2** [biigle/core#532] [Q&A]
I am annotating videos using BIIGLE and I am running into a problem with the playback function of all videos within a certain volume. The videos are displaying and playing, but when I pause the videos, they restart at 00:00. Any advice on how to fix this issue would be greatly appreciated. 

Thanks!

**Comment by mzur** [comment]
Do you happen to serve the videos from MS Azure? Please have a look at #471. Azure defaults to an older API version that does not support the HTTP `Range` header. This header is required for correct video playback.

**Reply by jcostel2** [reply]
I did serve the videos from MS Azure. I'll take a look at #471, Thanks
